extensions/omni_txn/tests/retry.yml

```diff
@@ -117,14 +117,14 @@ tests:
     - count: 1
       iso_level: serializable
 
-- name: max attempts
+- name: max attempts with timeout
```

Comment:
We want to test all relevant behaviors – max_attempts, timeouts, etc. – whether they follow the logic we decide on

Reply:
This likely means separate test for each case.